src/Compilers/CSharp/Portable/CSharpResources.resx

```diff
@@ -6870,28 +6870,43 @@ To remove the warning, you can use /reference instead (set the Embed Interop Typ
     <value>At least one top-level statement must be non-empty.</value>
   </data>
   <data name="ERR_LineDoesNotStartWithSameWhitespace" xml:space="preserve">
-    <value>Line does not start with the same whitespace as the last line of the raw string literal</value>
+    <value>Line does not start with the same whitespace as the last line of the raw string literal.</value>
   </data>
   <data name="ERR_RawStringNotInDirectives" xml:space="preserve">
-    <value>Raw string literals are not allowed in preprocessor directives</value>
+    <value>Raw string literals are not allowed in preprocessor directives.</value>
   </data>
   <data name="ERR_RawStringDelimiterOnOwnLine" xml:space="preserve">
-    <value>Raw string literal delimiter must be on its own line</value>
+    <value>Raw string literal delimiter must be on its own line.</value>
```

Comment:
For "Line does not start" and "Raw string literal delimiter": The surrounding error messages don't attempt to leave out `A` and `The`. I like the more casual form that uses articles and sounds like a person talking rather than an up-tight title of something that is missing as many words as possible, but mainly I'm thinking about consistency across the messages.